### PR TITLE
Excluding keys

### PR DESCRIPTION
**Pull request by LarsWiegers**
Introduces the possibility to exclude keys via config or parameter to the command.

### CODE REVIEW

File: config/translation-checker.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laravel Translation Checker
+    |--------------------------------------------------------------------------
+    |
+    | This package allows you to check your Laravel translations for missing
+    | keys, unused keys and unused translations.
+    | You can ignore some keys by adding them to the 'ignore' array.
```
Comment (jhm-ciberman) [inline config/translation-checker.php:11]: I know this is the description i used in my PR but currently the package let's you find missing keys but don't have any functionality related to "unused keys/translations". When I opened the PR I just pasted an example description without paying too much attention. 
Maybe you can remove that part to better reflect the current functionality of the package.

**Review by jhm-ciberman — APPROVED** [review]
I haven't tested this code in our repository, but I've checked the code changed and it looks fine to me. 

Thanks! 😁

**Comment by jhm-ciberman** [comment]
As a side comment about the code, unrelated to this PR:

I saw you use a double `for` loop to collect the `realLines` array used to compare. 

I'm reading the code from my phone so maybe I'm missing something but that would make the checking to work 2 levels deep, am I right?

Maybe you can use [`Arr:dot($arr)`](https://laravel.com/docs/10.x/helpers#method-array-dot) built in helper (and `Arr::undot`) to easily collect all the keys.

**Comment by LarsWiegers** [comment]
Closing as this will be handled in #42

**Pull request closed without merging**